ci/long_running_tests/README.rst

```diff
@@ -6,45 +6,39 @@ forever until they fail. To set up the project you need to run
 
 .. code-block:: bash
 
-    pip install anyscale
-    anyscale project create
+    $ pip install anyscale
+    $ anyscale init
 
 Note that all the long running test is running inside virtual environment, tensorflow_p36
 
 Running the Workloads
 ---------------------
+Easiest approach is to use the `Anyscale UI <https://www.anyscale.dev/>`. First run ``anyscale snapshot create`` from the command line to create a project snapshot. Then from the UI, you can launch an individual session and execute the run command for each test. 
 
-You can start all the workloads with:
+You can also start the workloads using the CLI with:
 
 .. code-block:: bash
 
-    anyscale session start -y run --workload="*" --wheel=https://s3-us-west-2.amazonaws.com/ray-wheels/releases/0.7.5/6da7eff4b20340f92d3fe1160df35caa68922a97/ray-0.7.5-cp36-cp36m-manylinux1_x86_64.whl
+    $ anyscale start
+    $ anyscale run run --workload=<WORKLOAD_NAME> --wheel=<RAY_WHEEL_LINK>
```

Comment:
run twice?

Reply:
The first one is for the `anyscale run` command, and the second one is the actual name of the command. But it is confusing so I changed the name of the command to `test_workload`. @rkooo567 can I also get your OK on this?